cypress/integration/pages/index.spec.ts

```diff
@@ -20,4 +20,24 @@ describe('Homepage', () => {
 
 		cy.get('h1').should('be.visible').should('contain', 'How much am I');
 	});
+
+	it('footer is visible', () => {
+		cy.visit('/');
+
+		cy.get('footer')
+			.should('be.visible')
+			.find('a[href="/faq"]')
+			.should('have.attr', 'href', '/faq')
+			.click();
+
+		cy.url().should('contain', 'faq');
+	});
+
+	it('can click on a city', () => {
+		cy.visit('/');
+
+		cy.get('.pt-2').find('a').first().click();
```

Comment:
`.pt-2` is not a good selector, it's tailwindCSS's way of saying "add some padding-top", and we might add some other "pt-2" elements here and there.

Have a read at https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements. I would recommend to add a `data-cy="city-card-1st"` on the 1st card element.

Reply:
Thanks, I wasn't aware of tailwind classes. I read it as "part 2" and assumed it was the second container or something. I didn't want to go ahead and add a data thing if possible but it seems like that's the best option. Will go ahead and add one and update the test selector.